pipeline/Plant_information.csv

```diff
@@ -1,48 +1,48 @@
+Carl Linnaeus,carl.linnaeus@lnhm.co.uk,(146)994-1635x35992,36.24624,139.07204,Fujioka,JP,Gaillardia,Gaillardia aestivalis,https://perenual.com/storage/image/upgrade_access.jpg,13.373955162620454,32.46962844577115,2024-11-26 09:09:31,"Mon, 25 Nov 2024 14:18:23 GMT"
+Eliza Andrews,eliza.andrews@lnhm.co.uk,(846)669-6651x75948,44.92801,4.8951,Valence,FR,Amaryllis,Hippeastrum (group),https://perenual.com/storage/image/upgrade_access.jpg,9.051587723276157,36.56766654343809,2024-11-26 09:09:31,"Mon, 25 Nov 2024 14:05:38 GMT"
+Carl Linnaeus,carl.linnaeus@lnhm.co.uk,(146)994-1635x35992,22.4711,88.1453,Pujali,IN,Caladium Bicolor,Caladium bicolor,https://perenual.com/storage/species_image/1457_caladium_bicolor/og/25575875658_d782fb76f1_b.jpg,11.717453923043404,34.833023257462685,2024-11-26 09:09:31,"Mon, 25 Nov 2024 13:45:13 GMT"
+Carl Linnaeus,carl.linnaeus@lnhm.co.uk,(146)994-1635x35992,41.57439,24.71204,Smolyan,BG,Chlorophytum Comosum,Chlorophytum comosum 'Vittatum',https://perenual.com/storage/species_image/1847_chlorophytum_comosum_vittatum/og/2560px-Chlorophytum_comosum_27Vittatum27_kz02.jpg,9.841879067582477,36.303309092664094,2024-11-26 09:09:32,"Mon, 25 Nov 2024 14:49:43 GMT"
+Gertrude Jekyll,gertrude.jekyll@lnhm.co.uk,001-481-273-3691x127,20.22816,-103.5687,Zacoalco de Torres,MX,Araucaria Heterophylla,Araucaria heterophylla,https://perenual.com/storage/species_image/917_araucaria_heterophylla/og/49833684212_2aff9d7b3c_b.jpg,9.540464188643528,32.68450523260249,2024-11-26 09:09:33,"Mon, 25 Nov 2024 13:32:40 GMT"
+Gertrude Jekyll,gertrude.jekyll@lnhm.co.uk,001-481-273-3691x127,33.95015,-118.03917,South Whittier,US,Begonia,Begonia 'Art Hodes',None,8.968570655798558,33.547627136363644,2024-11-26 09:09:34,"Mon, 25 Nov 2024 14:04:23 GMT"
+Eliza Andrews,eliza.andrews@lnhm.co.uk,(846)669-6651x75948,-13.7804,34.4587,Salima,MW,Medinilla Magnifica,Medinilla magnifica,https://perenual.com/storage/image/upgrade_access.jpg,13.288680852596213,37.18531383701657,2024-11-26 09:09:35,"Mon, 25 Nov 2024 14:12:39 GMT"
+Eliza Andrews,eliza.andrews@lnhm.co.uk,(846)669-6651x75948,37.49223,15.07041,Catania,IT,Calliandra Haematocephala,Calliandra haematocephala,https://perenual.com/storage/species_image/1477_calliandra_haematocephala/og/52063600268_834ebc0538_b.jpg,13.32643222871992,-1.08411359714286,2024-11-26 09:09:36,"Mon, 25 Nov 2024 13:30:18 GMT"
+Carl Linnaeus,carl.linnaeus@lnhm.co.uk,(146)994-1635x35992,14.14989,121.3152,Calauan,PH,Zamioculcas Zamiifolia,Zamioculcas zamiifolia,https://perenual.com/storage/image/upgrade_access.jpg,14.878464323084312,34.463350951171876,2024-11-26 09:09:38,"Mon, 25 Nov 2024 14:31:09 GMT"
+Eliza Andrews,eliza.andrews@lnhm.co.uk,(846)669-6651x75948,17.94979,-94.91386,Acayucan,MX,Crassula Ovata,Crassula ovata,https://perenual.com/storage/species_image/2193_crassula_ovata/og/33253726791_980c738a1e_b.jpg,9.354218607628031,33.09518671622935,2024-11-26 09:09:40,"Mon, 25 Nov 2024 14:02:15 GMT"

```

Comment:
can we add this file to the .gitignore pls! can always run extract before doing transform